### PR TITLE
[BACKPORTS][v1.0.0] - #2477 and #2479

### PR DESCRIPTION
**Pull request by seriva**
This pull is for backport issues #2477 and #2479. Did both of them in the same pull as some code was overlapping. Like in the linked related PR below, there is no need for reviewing all validation YAMLs as they should by mainly tested by @przemyslavic .

Issue: #2477
Backport for issue: #195, #2425
Related PR: #2470, #2474

issue: #2479
Backport for issue: #1294, #1370
Related PR: #2465

Also included issue #1296 with related PR #2552, as there where a minor addition for #2425

### CODE REVIEW

**Comment by rafzei** [comment]
Should we hang on with review/merge till @przemyslavic will be able to start testing ? (nobody wants to break a LTS branch :cactus: )

**Comment by seriva** [comment]
> Should we hang on with review/merge till @przemyslavic will be able to start testing ? (nobody wants to break a LTS branch 🌵 )

It can be reviewed but as you said let @przemyslavic test it first before merging to LTS.

**Review by erzetpe — DISMISSED** [review]

**Review by rafzei — DISMISSED** [review]

**Comment by przemyslavic** [comment]
Additional fix required https://github.com/epiphany-platform/epiphany/issues/1294#issuecomment-900194372.

**Review by erzetpe — DISMISSED** [review]

**Review by rafzei — DISMISSED** [review]

**Review by rafzei — DISMISSED** [review]

**Review by erzetpe — DISMISSED** [review]

**Review by rafzei — DISMISSED** [review]

**Comment by przemyslavic** [comment]
@seriva I think we should update all validations for v1.0. For example, the `use_repmgr` property is missing for PostgreSQL config. In defaults there is also a parameter with a typo `priviledged_user_name`, and in validation it is already correct (`privileged_user_name`), but it means that validation will not check this field.
Same for `specification.clustered` which is missing from logging component.
I suppose there are more such places.

**Comment by seriva** [comment]
> @seriva I think we should update all validations for v1.0. For example, the `use_repmgr` property is missing for PostgreSQL config. In defaults there is also a parameter with a typo `priviledged_user_name`, and in validation it is already correct (`privileged_user_name`), but it means that validation will not check this field. Same for `specification.clustered` which is missing from logging component. I suppose there are more such places.

Good point, ill rerun the validation process for all documents tomorrow on both branches. Should be like an hour.

**Comment by seriva** [comment]
@przemyslavic  Fixed a bunch of minor issues across multiple validation files.

/azp run

File: core/src/epicli/data/common/validation/configuration/grafana.yml [hunk 1]
```diff
@@ -335,16 +323,15 @@ properties:
       provider:
         type: string
       key_file:
-        type: 'null'
+        type: string
       bucket:
-        type: 'null'
+        type: string'
```
Comment (przemyslavic) [inline core/src/epicli/data/common/validation/configuration/grafana.yml]: string' 🤔

Comment (seriva) [reply core/src/epicli/data/common/validation/configuration/grafana.yml]: A test to check your alertness, you passed.

**Review by przemyslavic — APPROVED** [review]

**Review by sbbroot — APPROVED** [review]

**Pull request merged**